src/test/resources/minimal-epub/OPS/package.opf

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" version="3.0" xml:lang="en" unique-identifier="q">
+<metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+  <dc:title id="title">Minimal EPUB</dc:title>
+  <dc:language>en</dc:language>
+  <dc:identifier id="q">NOID</dc:identifier>
+  <meta property="dcterms:modified">2017-06-12T00:00:01Z</meta>
+</metadata>
+<manifest>
+  <item id="content_001"  href="content_001.html" media-type="application/xhtml+xml"/>
```

Comment:
content docs [should have the extension `.xhtml`](http://www.idpf.org/epub/31/spec/epub-contentdocs.html#confreq-cd-xhtml-fileprops-name) (2017, right?)